proposals/stage-2/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -1,55 +1,49 @@
-# Project Name
+# ${Project Name}
 
 ## Project Champion
 
 Who is the main point of contact during the application process?
 
-## CPC Champion (optional)
-
-If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
-
 ## Description
 
-A rough description of the project
+A rough description of the project in less than 100 words.
 
 ## Statement of alignment with OpenJS Foundation charter and mission
 
-Please refer to [TBD][]
-
-## Current Code of Conduct
+Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
+Keep answers to less than 250 words.
 
-Include a link
+## List of all repos that are part of the project
 
-## Current Project License
-
-Include a link
-
-## Current Source Control
-
-Link to Github or other source control
+Include links and license information
 
 ## Issues Tracker
 
 Link to GitHub or other tracker
```

Comment:
If issue trackers are per repo, do you want the whole list too? Wouldn't it make more sense to group all the per repo info you need in a single place?

Reply:
That would make sense. Wanna take a stab at iteration on text?